### PR TITLE
actually implement oh no

### PR DESCRIPTION
**Pull request by BasedUser**
not yet complete
TODO: implement missingtext code so that i don't need to update this every build

### CODE REVIEW

File: core/assets/bundles/bundle_oh_NO.properties [hunk 1]
```diff
@@ -0,0 +1,1136 @@
...
+mech.delta-mech.description = [scarlet]oh no
+mech.tau-mech.description = [scarlet]oh no
+mech.omega-mech.description = [scarlet]oh no
+mech.dart-ship.description = [scarlet]oh no
+mech.javelin-ship.description = [scarlet]oh no
+mech.trident-ship.description = [scarlet]oh no
+mech.glaive-ship.description = [scarlet]oh no
+unit.draug.description = [scarlet]oh no
+unit.spirit.description = [scarlet]oh no
+unit.phantom.description = An advanced drone unit. Follows users. Assists in block construction.
```
Comment (Fayti1703) [inline core/assets/bundles/bundle_oh_NO.properties]: That's not an oh no.

**Comment by Arkanic** [comment]
yes.

File: core/assets/bundles/bundle_oh_NO.properties (hunk 1 above)
Comment (Arkanic) [reply core/assets/bundles/bundle_oh_NO.properties]: oh no

Comment (BasedUser) [reply core/assets/bundles/bundle_oh_NO.properties]: I need to update the bundle to include steam text anyways

**Comment by BasedUser** [comment]
heres a thought, what if i actually maintain the oh no translation while i figure out a way to trick arc?

**Comment by Anuken** [comment]
Closing until a better solution is found - I really don't want to have to maintain/merge PRs just to keep this bundle updated.

**Comment by GlennFolker** [comment]
oh no

**Pull request closed without merging**